src/OrchardCore.Modules/OrchardCore.Security/Services/HeaderPolicyProvider.cs

```diff
@@ -7,6 +7,11 @@ public abstract class HeaderPolicyProvider : IHeaderPolicyProvider
     {
         public SecurityHeadersOptions Options { get; set; }
 
+        public virtual void InitializePolicy()
+        {
+
+        }
```

Comment:
And comment why it's intentionally blank.

```suggestion
        public virtual void InitializePolicy()
        {
        }
```

Reply:
There's no default implementation, also this is useful only the headers the require initialization can override it

Reply:
Add a comment to the code, please ;).